tests/test_default_headers.py

```diff
@@ -36,6 +36,20 @@ async def test_override_server_header():
             )
 
 
+@pytest.mark.asyncio
+async def test_disable_default_server_header():
+    config = Config(
+        app=App,
+        loop="asyncio",
+        limit_max_requests=1,
+        default_server_header=False,
+    )
+    async with run_server(config):
+        async with httpx.AsyncClient() as client:
+            response = await client.get("http://127.0.0.1:8000")
+            assert "server" not in response.headers
```

Comment:
shouldn't it be `assert "uvicorn" not in response.headers` ?

Reply:
The value is "uvicorn", but the key is "server".

Reply:
mmm my bad :facepalm: